src/solc_0.7/catalyst/AssetAttributesRegistry.sol

```diff
@@ -4,20 +4,10 @@ pragma experimental ABIEncoderV2;
 
```

Comment:
Same here : revert back the formating to follow the same layout.
keeping the minimal changes